src/components/text/_links.scss

```diff
@@ -0,0 +1,40 @@
+.mint-link {
+  @include remove-descenders;
+  cursor: pointer;
```

Comment:
I'm not sure about that one - links natively have `cursor: pointer`, so this setting only makes sense if we create something that should look and feel like link but should not be an `<a>`. Do we have such cases?

Anyway, this works on opera mini (tested in footer where we have links now), so that aspect is not an issue.